src/main/java/com/user_service/DTOs/ConfirmForgotPasswordDto.java

```diff
@@ -0,0 +1,10 @@
+package com.user_service.DTOs;
+
+import lombok.Data;
+
+@Data
```

Comment:
Any specific reason why this is a mutable class? Personally for DTOs I would go with immmutables by default https://projectlombok.org/features/Value. This comment applies to other DTO classes too.